src/Store.js

```diff
@@ -435,8 +434,8 @@ class Store {
    * @return {Array} array of objects
    */
   getRecords (type) {
-    const records = Object.values(this.getType(type).records)
-                          .filter(value => value && value !== 'undefined')
+    const records = Array.from(this.getType(type).records.values())
```

Comment:
.values() returns an iterator so we need to convert it to an array to chain .filter